config

```diff
@@ -663,7 +663,7 @@ case "$GUESSOS" in
   *-*-linux2) OUT="linux-generic32" ;;
   sun4[uv]*-*-solaris2)
 	OUT="solaris-sparcv9-$CC"
-	ISA64=`(isalist) 2>/dev/null | grep sparcv9`
+	ISA64=`(isainfo) 2>/dev/null | grep sparcv9`
```

Comment:
isainfo(1) was introduced in Solaris 7 in the late 90s, so it should be safe to just replace isalist(1) with isainfo(1).